### PR TITLE
Add policy packs for IAM service

### PR DESCRIPTION
**Pull request by RahulLah19**
### Policy set by smart folder
image
image

### Control state
image

### CODE REVIEW

File: hub_content/policy_packs/aws/iam/enforce_disallow_configuration_changes/policies.tf [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        Id
+        metadata
+      }
+    }
+    EOT
+  template       = <<-EOT
+    |
+    resource "aws_iam_policy" "deny_config_changes" {
+      name        = "DenyConfigChanges"
+      description = "Deny specific Config actions unless the principal is AWSControlTowerExecution role"
```
Comment (SumitPopat) [inline hub_content/policy_packs/aws/iam/enforce_disallow_configuration_changes/policies.tf:25]:
```suggestion
      name        = "deny_aws_config_changes"
      description = "Disallow configuration changes to AWS config"
```

File: hub_content/policy_packs/aws/iam/enforce_disallow_configuration_changes/policies.tf [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    {
+      account  {
+        Id
+        metadata
+      }
+    }
+    EOT
+  template       = <<-EOT
+    |
+    resource "aws_iam_policy" "deny_config_changes" {
```
Comment (SumitPopat) [inline hub_content/policy_packs/aws/iam/enforce_disallow_configuration_changes/policies.tf:23]:
```suggestion
    resource "aws_iam_policy" "deny_aws_config_changes" {
```

File: hub_content/policy_packs/aws/iam/enforce_disallow_configuration_changes/common.tf [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Policy Pack
+resource "turbot_smart_folder" "pack" {
+  title       = "Disallow Configuration Changes to AWS Config"
+  description = "IAM policy to disallow configuration change in AWS config"
```
Comment (SumitPopat) [inline hub_content/policy_packs/aws/iam/enforce_disallow_configuration_changes/common.tf:4]:
```suggestion
  title       = "Deny configuration changes to AWS config"
  description = "Disallow configuration changes to AWS config."
```

File: hub_content/policy_packs/aws/iam/enforce_disallow_configuration_changes/common.tf
Comment (SumitPopat) [inline hub_content/policy_packs/aws/iam/enforce_disallow_configuration_changes/common.tf:1]: I think the folder name should be `deny_changes_to_aws_config`

Comment (RahulLah19) [reply hub_content/policy_packs/aws/iam/enforce_disallow_configuration_changes/common.tf:1]: sounds good and to the point.

**Review by SumitPopat — APPROVED** [review]

**Pull request merged**